helmfile/values/harbor.yaml.gotmpl

```diff
@@ -88,6 +88,7 @@ core:
   nodeSelector: {{- toYaml .Values.harbor.nodeSelector | nindent 4 }}
   affinity:     {{- toYaml .Values.harbor.affinity | nindent 4 }}
   tolerations:  {{- toYaml .Values.harbor.tolerations | nindent 4 }}
+  topologySpreadConstraints: {{- toYaml .Values.harbor.core.topologySpreadConstraints | nindent 4  }}
```

Comment:
I can't seem to find that these are supported in harbor? Did you verify that these worked as expected?

Reply:
Right harbor didn't have support for this, I removed it now.